app/connector/mongodb/pom.xml

```diff
@@ -83,6 +83,10 @@
       <groupId>io.syndesis.connector</groupId>
       <artifactId>connector-support-util</artifactId>
     </dependency>
+    <dependency>
+      <groupId>io.syndesis.common</groupId>
+      <artifactId>common-util</artifactId>
```

Comment:
We should not depend on Syndesis modules in the connectors, that is everything but the integration-runtime. I know we have some connectors that transgress on this but we should try not to introduce new dependencies like this.

This prevents us from using different dependencies at integration runtime and in the Syndesis server.

Reply:
Actually 20 over 33 connectors are using this utility dependency. It's handy for [metadata handling](https://github.com/syndesisio/syndesis/blob/master/app/common/util/src/main/java/io/syndesis/common/util/json/JsonUtils.java). I will rework that part, but we should consider the effort of refactoring the rest.

Reply:
Yeah, we need to refactor that, let's not make it more difficult on ourselves and add more things we need to refactor. See if you can omit it and then we can apply the same pattern to the rest of the connectors.

We have had issues (#6948) upgrading the version of Jackson and than having a different version of at runtime, dictated by Camel BOM/Spring Boot BOM or the integration-runtime BOM.

This will be an ongoing effort I think trying to separate the dependency trees of integrations at runtime and the dependencies we use in the server.